assets/src/components/conversations/ConversationsProvider.tsx

```diff
@@ -289,9 +289,7 @@ export class ConversationsProvider extends React.Component<Props, State> {
 
     this.channel.push('shout', {
       body: message,
-      user_id: userId,
       conversation_id: conversationId,
-      account_id: accountId,
```

Comment:
(these values are no longer necessary since they can be inferred from the auth token)

Reply:
yup exactly